src/clj/rems/home.clj

```diff
@@ -40,11 +48,15 @@
   (GET "/" []
     (layout/home-page))
 
+  ;; TODO should these redirects have swagger documentation?
```

Comment:
I suppose it's not very easy unless they are moved under `/api`? I think https://github.com/CSCfi/rems/blob/master/docs/linking.md is enough.

Reply:
The ring wiki says it's a vector. https://github.com/ring-clojure/ring/wiki/Parameters

Reply:
Yeah it's probably enough, but it would  be nice to document these (and e.g. the exact parameter formats and HTTP return codes) as swagger. We might be able to merge two swagger docs or something? But maybe let's not worry about it now.